draft-ietf-oauth-status-list.md

```diff
@@ -648,6 +649,11 @@ IANA "CBOR Web Token (CWT) Claims" registry {{IANA.CWT}} established by {{RFC839
 *  Change Controller: IETF
 *  Specification Document(s):  [](#status-list-token-cwt) of this specification
 
+* Claim Name: `ttl`
+* Claim Description: Time to Live
```

Comment:
As discussed on the call lets add the Claim Key entry to each IANA registration with the proposed values.